fedimint-bin-tests/src/main.rs

```diff
@@ -525,26 +571,27 @@ struct DevFed {
     gw_lnd: Gatewayd,
     electrs: Electrs,
     esplora: Esplora,
+    process_mgr: ProcessManager,
```

Comment:
no, don't add process_mgr here.

just pass it to your function as a argument function.

Reply:
I initially did that, I think it makes more sense to have it in `DevFed`. Can you tell me why is it better to be passed as argument?

Reply:
sorry it didn't explain
in my head, process manager is like a pseudo global, no one owns it. 

I think it is fine to put process manager to be in devfed, but I would prefer not to.

Reply:
Oh okay no problem, i'll pass it as argument.
In my head, devfed was like a controller, with tools to start/stop deamons.